### DISCUSSION TITLE
I need help prefilling an embedded Cognito form

### DISCUSSION THREAD

**Discussion by Marcelle229** [community/community#131877] [Programming Help]
### Select Topic Area

Question

### Body

Please see my full question in Stack overflow: https://stackoverflow.com/questions/78733167/im-trying-to-prefill-2-cognito-forms-that-are-embedded-in-a-website-using-url-pa

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @Marcelle229 ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
